api/inc/svc_exports.h

```diff
@@ -116,6 +116,7 @@
 /* SVC immediate values for hardcoded table (call from unprivileged) */
 #define UVISOR_SVC_ID_UNVIC_OUT        UVISOR_SVC_FIXED_TABLE(0, 0)
 #define UVISOR_SVC_ID_REGISTER_GATEWAY UVISOR_SVC_FIXED_TABLE(3, 0)
+#define UVISOR_SVC_ID_BOX_INIT_NEXT    UVISOR_SVC_FIXED_TABLE(5, 0)
```

Comment:
This was done to start a discussion, but it looks like the XXX was missing.

What were 1 and 2? Those were skipped, so maybe I should skip 4 for similar reasons.